flytepropeller/pkg/controller/nodes/subworkflow/subworkflow.go

```diff
@@ -88,6 +88,25 @@ func (s *subworkflowHandler) handleSubWorkflow(ctx context.Context, nCtx interfa
 		return handler.DoTransition(handler.TransitionTypeEphemeral, handler.PhaseInfoRunning(nil)), nil
 	}
 
+	if state.HasTimedOut() {
```

Comment:
Can you confirm this works with the workflow failure policy set to execute as many nodes as possible? Just want to make sure we're covering this if we weren't previously.

Reply:
Just tested setting `failure_policy=WorkflowFailurePolicy.FAIL_AFTER_EXECUTABLE_NODES_COMPLETE`
and it works as expected. Updated the screenshot above showing another task in the subworkflow completing after a task times out.

Reply:
perfect, thanks!